WordPress/Classes/ViewRelated/Reader/Cards/ReaderPostCell.swift

```diff
@@ -271,14 +272,31 @@ private final class ReaderPostCellView: UIView {
         detailsLabel.text = viewModel.details
 
         imageView.isHidden = viewModel.imageURL == nil
+
         if let imageURL = viewModel.imageURL {
-            imageView.setImage(with: imageURL)
+            imageView.setImage(with: imageURL, size: preferredCoverSize)
         }
 
         configureToolbar(with: viewModel.toolbar)
         configureToolbarAccessibility(with: viewModel.toolbar)
     }
 
+    private lazy var mainWindow = UIApplication.shared.mainWindow
+
+    private var preferredCoverSize: CGSize? {
+        guard let mainWindow else { return nil }
```

Comment:
Using `self.window` instead?

Reply:
What do you think about using `let window = self.window ?? UIApplication.shared.mainWindow`? I don't see the need to store `UIApplication.shared.mainWindow`?

Reply:
Sure, I think it makes sense no to retain global fields. Updated.